script.js

```diff
@@ -356,4 +356,47 @@ form.addEventListener('submit', (e) => {
     return false;
```

Comment:
[OPTIONAL]
- [ ] Kindly make sure you use the correct validation for the email field, as you may notice there's no upper case in here but I get this error.
`Hint: You may consider using: `
```js
// the regex expression
/^[^A-Z]+$/
// or comparing the email value to its corresponding lowercase like
email.value == email.value.toLowerCase
// Or switch around your logic in the if-else statement
```
![image](https://user-images.githubusercontent.com/97100091/211026452-9cf6bb42-830a-4d63-8534-150504688d40.png)